.changeset/perfect-paws-explode.md

```diff
@@ -2,4 +2,4 @@
 'xstate': patch
 ---
 
-Fixed an issue with some exit handlers being executed more than once when stopping a machine.
+Fixed an issue with some exit actions being executed more than once when stopping a machine.
```

Comment:
I've just tweaked this as usually we don't call those 'handlers" in the docs so this could be confusing for people